src/renderer/services/file-management-system/index.ts

```diff
@@ -485,15 +533,16 @@ export default class FileManagementSystem {
    * Uploads the given file to FSS in chunks asynchronously using a NodeJS.
    */
   private async uploadInChunks(config: {
-    fssUploadId: string,
+    fssStatus: UploadStatusResponse,
     source: string,
-    chunkSize: number,
     user: string,
     onProgress: (bytesUploaded: number) => void,
     initialChunkNumber?: number,
     partiallyCalculatedMd5?: string,
   }): Promise<void> {
-    const { fssUploadId, source, chunkSize, user, onProgress, initialChunkNumber = 0, partiallyCalculatedMd5 } = config;
+    const { fssStatus, source, user, onProgress, initialChunkNumber = 0, partiallyCalculatedMd5 } = config;
+    const fssUploadId = fssStatus.uploadId;
+    const chunkSize = fssStatus.chunkSize;
```

Comment:
Is this chunkSize guaranteed to be the chunkSize at the time of the original upload's start? Like if we upgrade FSS after an upload has been attempted could the chunkSize change? That is the reasoning behind why it previously relied on the serviceFields. If we can just rely on FSS, lets remove it from the serviceField if possible.

Reply:
I can't think of anything too terrible happening.  The only reasons I know or that chunkSize matters are:
- performance
- keeping the total number of chunks below the S3 limit.

If we make a perf related change on the backend, I dont see an upload resuming at the new size being a problem.  If it is related to the S3 limit, then something must have changed with S3 usage and continuing under the old assumption seems unwise.  What do you think?

Reply:
It would be a problem due to how we calculate the offset of where we left off in the upload. If we uploaded 5 500mb chunks (2500mb) then change the chunksize to 800 on the backend we would resume at 4000mb

Reply:
Actually, the chunksize being passed in here is from the the FSS status, not the service field.

Reply:
>Actually, on main the chunksize being passed in here is also from the the FSS status, not the service field.

Ah so its been a potential problem for a while, could be too niche to matter then

Reply:
backlog ticket

Reply:
[SWE-879](https://aicsjira.corp.alleninstitute.org/browse/SWE-879)